src/run.jl

```diff
@@ -77,9 +77,18 @@ function build_input(tensor_map::Dict)
     input_tensors, input_values
 end
 
+struct ClosedSessionError <: Exception
+end
+
+function Base.show(io::IO, err::ClosedSessionError)
+    print(io, "Tried to call 'run' with a closed TensorFlow session.")
```

Comment:
This is assuming the call-stack.
Right now this can only be thrown by `run`, true.
But the user will find that out by looking at the call-stack.

I think it should be "The TensorFlow session is closed.")

Reply:
Good point